src/LanguageServer/Test/MissingImportCodeActionTests.cs

```diff
@@ -503,7 +521,7 @@ import socket
             // calculate actions
             var diagnosticSpan = spans["diagnostic"].First().ToSourceSpan(analysis.Ast);
             var diagnostics = GetDiagnostics(analysis, diagnosticSpan, MissingImportCodeActionProvider.Instance.FixableDiagnostics);
-            var codeActions = await new CodeActionSource(analysis.ExpressionEvaluator.Services).GetCodeActionsAsync(analysis, diagnostics, CancellationToken.None);
+            var codeActions = await new QuickFixCodeActionSource(analysis.ExpressionEvaluator.Services).GetCodeActionsAsync(analysis, CodeActionSettings.Default, diagnostics, CancellationToken.None);
 
```

Comment:
Should probably pass reasonable cancellation token

Reply:
Oh, nvm its test code

Reply:
any example on existing test?

Reply:
found timeout attribute supported by mstest. made test to use it.